crates/gpui/src/platform/linux/platform.rs

```diff
@@ -384,9 +384,20 @@ impl<P: LinuxClient + 'static> Platform for P {
         self.set_cursor_style(style)
     }
 
-    // todo(linux)
     fn should_auto_hide_scrollbars(&self) -> bool {
-        false
+        let cmd = Command::new("gsettings")
+            .arg("get")
+            .arg("org.gnome.desktop.interface")
+            .arg("overlay-scrolling")
+            .output()
+            .unwrap();
```

Comment:
In addition to what @apricotbucket28 was saying, we don't want to unwrap in case this process fails.

Reply:
Ok